test/CMakeLists.txt

```
@@ -249,5 +249,13 @@ celeritas_setup_tests(SERIAL PREFIX field)
 
 celeritas_cudaoptional_test(field/RungeKutta)
 
+if(CELERITAS_USE_CUDA)
+  add_library(celeritas_field_test field/FieldDriver.test.cu)
+  target_link_libraries(celeritas_field_test PRIVATE celeritas)
+  list(APPEND CELERITASTEST_LINK_LIBRARIES celeritas_field_test)
+endif()
+
+celeritas_add_test(field/FieldDriver.test.cc GPU)
+
```

You  should be able to do

```suggestion
celeritas_cudaoptional_test(field/FieldDriver)
```
if you set up the cuda-disabled cases like the `RungeKutta` tests do, with inline `#if !CELERITAS_USE_CUDA` methods in the `.hh` file and a `TEST_IF_CELERITAS_CUDA` macro in the `.cc` file.